src/docs-style-guide.rst

```diff
@@ -25,7 +25,28 @@ Whenever U.S. English and British (or other) English spelling or usage disagree,
 .. code-block:: rst
 
   The color of the button is gray.
+
+.. code-block:: python
+  :class: proselint-extra-checks
   
+  @memoize
```

Comment:
Something here should indicate that this is an excerpt, with some details on tracking down the original.

(This could be in code comments in the Python block, at the point of truncation.)